src/robot/handler/protobuf_test.py

```diff
@@ -2,12 +2,43 @@
 Test for protobuf handler controller
 """
 import pytest
-from proto.gen.pb_pb2 import BusMessage, LaserSensorMsg, PressureSensorMsg
+
+from proto.gen.outech_pb2 import EncoderPositionMsg, BusMessage, LaserSensorMsg, PressureSensorMsg
 from src.robot.handler.protobuf import ProtobufHandler
 
 
+@pytest.fixture(name='protobuf_handler')
+def protobuf_handler_setup(match_action_controller_mock,
+                           localization_controller_mock):
+    """
+    Setup a handler for testing.
+    """
+    return ProtobufHandler(
+        match_action_controller=match_action_controller_mock,
+        localization_controller=localization_controller_mock,
+    )
+
+
+@pytest.mark.asyncio
+async def test_dispatch_encoder_position(protobuf_handler,
+                                         localization_controller_mock):
+    """
+    Dispatch encoder position to localization controller.
+    """
+    bus_message = BusMessage(encoderPosition=EncoderPositionMsg(
+        left_tick=1,
+        right_tick=-2,
+    ))
+    msg_bytes = bus_message.SerializeToString()
+    await protobuf_handler.translate_message(msg_bytes)
+
+    localization_controller_mock.update_odometry_position.assert_called_once_with(
+        1, -2)
```

Comment:
+ test with left-tick or right_tick or both missing ?

Reply:
It's not possible